src/GovUk.Education.ExploreEducationStatistics.Content.Services.Tests/PublicationServiceTests.cs

```diff
@@ -288,6 +288,168 @@ await using (var contentDbContext = InMemoryContentDbContext(contentDbContextId)
             }
         }
 
+        [Fact]
+        public async Task Get_IsSuperseded_SupersededPublicationBuildsPublicationSupersededByViewModel()
+        {
+            var releaseId = Guid.NewGuid();
+            var publication = new Publication
+            {
+                Title = "Publication Title",
+                Slug = PublicationSlug,
+                LatestPublishedReleaseId = releaseId,
+                SupersededBy = new Publication
+                {
+                    Id = Guid.NewGuid(),
+                    Title = "publication B",
+                    Slug = "publication-b",
+                    LatestPublishedReleaseId = Guid.NewGuid()
+                },
+                Releases = new List<Release>
+                {
+                    new()
+                    {
+                        Id = releaseId,
+                        ReleaseName = "2000",
+                        Slug = "2000",
+                        TimePeriodCoverage = TimeIdentifier.AcademicYear,
+                        Published = DateTime.UtcNow
+                    },
+                },
+                Topic = new Topic
+                {
+                    Title = "Test topic",
+                    Slug = "test-topic",
+                    Theme = new Theme
+                    {
+                        Title = "Test theme",
+                        Slug = "test-theme",
+                    }
+                },
+                Contact = new Contact
+                {
+                    TeamName = "Team name",
+                    TeamEmail = "team@email.com",
+                    ContactName = "Contact name",
+                    ContactTelNo = "1234",
+                },
+            };
+
+            var contentDbContextId = Guid.NewGuid().ToString();
+            await using (var contentDbContext = InMemoryContentDbContext(contentDbContextId))
+            {
+                await contentDbContext.AddAsync(publication);
+                await contentDbContext.SaveChangesAsync();
+            }
+
+            await using (var contentDbContext = InMemoryContentDbContext(contentDbContextId))
+            {
+                var service = SetupPublicationService(contentDbContext);
+                
+                var result = await service.Get(publication.Slug);
+
+                var publicationViewModel = result.AssertRight();
+                
+                Assert.Equal(publication.Id, publicationViewModel.Id);
+                Assert.True(publicationViewModel.IsSuperseded);
+                
+                Assert.Equal(publication.SupersededBy.Title, publicationViewModel.SupersededBy?.Title);
+                Assert.Equal(publication.SupersededBy.Slug, publicationViewModel.SupersededBy?.Slug);
+                Assert.Equal(publication.SupersededBy.Id, publicationViewModel.SupersededBy!.Id);
```

Comment:
I believe `Get_IsSuperseded_SupersedingPublicationHasNoPublishedRelease ` wouldn't be suitable for testing as `isSuperseded` will be set to false and `SupersededBy` will be null since you need to have a minimum of one published release in order for a publication to be superseded

Reply:
I see, thanks. I was assuming you were populating the view model based on the presence of `publication.SupersededBy` rather than the value of `IsSuperseded`. The two aren't necessarily in sync because as long as you've loaded `publication.SupersededBy` it can be not-null when `IsSuperseded` is false.

I think you can update `Get_IsSuperseded_SupersedingPublicationHasNoPublishedRelease` to add an assertion like `Assert.Null(publicationViewModel.SupersededBy)` to make sure it's not being populated.